internal/utils/metrics.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	invalidDataPoints = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "rosocp_invalid_datapoints_total",
+		Help: "The total number of invalid datapoints(rows) found in CSVs recevied",
```

Comment:
nit:
```suggestion
		Name: "rosocp_total_invalid_datapoints",
		Help: "The total number of invalid datapoints(rows) found in received CSVs",
```

Reply:
According to prometheus metric naming convention - https://prometheus.io/docs/practices/naming/ suffix should describe the unit.